src/api/private/auth/auth.controller.ts

```diff
@@ -49,55 +52,38 @@ export class AuthController {
 
   @UseGuards(RegistrationEnabledGuard)
```

Comment:
This guard can throw a BadRequestException which should get an ApiBadRequestResponse decorator.

Reply:
fixed